enhancements/samples-generation.md

```diff
@@ -0,0 +1,613 @@
+---
+title: Automatic Samples Generation using Go language
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-09-02
+last-updated: 2020-09-20
+status: implementable
+---
+
+# Automatic Samples Generation using Go language
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+1. Should not the project has a Memcached GO Sample with webhooks and other without using webhooks since it shows an advanced feature and it used is not mandatory? Note the Go tutorial do not use webhooks.  
+
+Answer: It should, since the memcached project is a sample for users to reference and webhooks are commonly used.
+
+## Summary
+
+The [operator-sdk-samples](https://github.com/operator-framework/operator-sdk-samples) repo contains an example Go operator project for users to reference when building their own projects. These example becomes outdated quickly because it requires its own release process and manual updates.
```

Comment:
```suggestion
The [operator-sdk-samples](https://github.com/operator-framework/operator-sdk-samples) repo contains an example Go operator project for users to reference when building their own projects. These examples become outdated quickly because it requires its own release process and manual updates.
```